src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/ActionButtons.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import {
+  Col, Row, Button, Popover,
+} from 'antd';
+import { useHistory, useLocation } from 'react-router-dom';
+import UseHandleRedirectToLoginClick from './DownloadUtils/UseHandleRedirectToLoginClick';
+import HandleDownloadManifestClick from './DownloadUtils/HandleDownloadManifestClick';
+import DownloadAllModal from './DownloadAllModal/DownloadAllModal';
+import DownloadAllFiles from './DownloadUtils/DownloadAllFiles/DownloadAllFiles';
+import DownloadJsonFile from './DownloadUtils/DownloadJsonFile';
+import { DiscoveryConfig } from '../../../DiscoveryConfig';
+import './ActionButtons.css';
+
+interface ActionButtonsProps {
+  isUserLoggedIn: boolean;
+  discoveryConfig: DiscoveryConfig;
+  resourceInfo: any;
+  healLoginNeeded: boolean;
+  noData: boolean;
+}
+
+const ActionButtons = ({
+  isUserLoggedIn,
+  discoveryConfig,
+  resourceInfo,
+  healLoginNeeded,
+  noData,
+}: ActionButtonsProps): JSX.Element => {
+  const { HandleRedirectToLoginClick } = UseHandleRedirectToLoginClick();
+  const [downloadStatus, setDownloadStatus] = useState({
+    inProgress: false,
+    message: { title: '', content: <React.Fragment />, active: false },
+  });
+  const history = useHistory();
+  const location = useLocation();
+  const showDownloadStudyLevelMetadataButtons: boolean | undefined = discoveryConfig?.features.exportToWorkspace.studyMetadataFieldName
+    && discoveryConfig?.features.exportToWorkspace.enableDownloadStudyMetadata
+    && resourceInfo?.study_metadata;
```

Comment:
instead of hard coding `study_metadata`, this should be the value of `discoveryConfig?.features.exportToWorkspace.studyMetadataFieldName`

Reply:
I updated this to reference this variable: 
```
  const studyMetadataFieldNameReference: string | undefined = discoveryConfig?.features.exportToWorkspace.studyMetadataFieldName;

```

In this manner: 
```
  const showDownloadStudyLevelMetadataButtons = Boolean(
    discoveryConfig?.features.exportToWorkspace.enableDownloadStudyMetadata
      && studyMetadataFieldNameReference
      && resourceInfo?.[studyMetadataFieldNameReference],
  );
  ```